tests/integration/team_test.py

```diff
@@ -42,13 +48,27 @@
 )
 
 
-@pytest.mark.xfail(reason="https://bugzilla.redhat.com/1798947")
-def test_create_team_iface():
+@pytest.mark.xfail(
+    raises=NmstateLibnmError, reason="https://bugzilla.redhat.com/1798947"
+)
+def test_create_team_iface_without_slaves():
     with team_interface(TEAM0) as team_state:
         assertlib.assert_state(team_state)
 
 
-@pytest.mark.xfail(reason="https://bugzilla.redhat.com/1798947")
+@pytest.mark.xfail(
+    raises=NmstateLibnmError, reason="https://bugzilla.redhat.com/1798947"
+)
+def test_create_team_iface_with_slaves():
+    with team_interface(TEAM0, [PORT1, PORT2]) as team_state:
+        assertlib.assert_state(team_state)
+        assert [PORT1, PORT2] == _get_runtime_team_slaves(TEAM0)
```

Comment:
This would not be needed with https://github.com/nmstate/nmstate/pull/799. But I am fine if you decide to use it anyway. Thanks!

Reply:
The code you use in #799  might not able to reflect the run time status. So checking with daemon is the most accurate way.

Reply:
We need to discuss this but in any case, I am fine with checking with the daemon. Thanks!